packages/website/src/components/EditorTabs.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import type { TabType } from './types';
+import styles from './Playground.module.css';
+import EditIcon from '@site/src/icons/edit.svg';
+
+export interface FileTabsProps {
+  readonly tabs: TabType[];
+  readonly activeTab: TabType;
+  readonly change: (tab: TabType) => void;
+  readonly showModal: () => void;
+}
+
+export default function EditorTabs({
+  tabs,
+  activeTab,
+  change,
+  showModal,
+}: FileTabsProps): JSX.Element {
```

Comment:
A few missing tab attributes around this component for accessibility...

* The div holding the buttons should have `role="tablist"`
* The buttons themselves should have `role="tab"`, `aria-selected`, and `aria-controls` props
* The openable components should have `role="tabpanel"`

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role

Reply:
`aria-controls` and `role="tabpanel"` doesn't apply in here, we actually have single content that is changes what is displayed

Reply:
Hmm, I don't think I follow where you're coming from. It's a set of buttons, each of which have an onclick that changes the content in the page. That's a pretty canonical use for tabs, no?

From https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role#frame_example:

![Screenshot of the three tabbed frame example from the MDN page](https://user-images.githubusercontent.com/3335181/170893057-40c506b3-7cc7-4f75-a0cf-4d531c464319.png)

Reply:
ok, i added role tab, tablist and `aria-selected` let me know if that's correct now

Reply:
Super, thanks!

I think there's more work that _could_ be done around `aria-controls`, but I'm not super familiar with this myself and don't want to block now that the basics are done.